practices/follow-functional-core-imperative-shell.md

```diff
@@ -3,117 +3,153 @@
+By communicating between the functional core and imperative shell via passing values instead of relying on interfaces, developers can mitigate the risk of false positives when using mocks and avoid encountering errors with the real implementation.
 
-In dynamic languages, the Functional Core, Imperative Shell approach is often favored due to its ability to address testing challenges inherent in these languages.
-By communicating between the Functional Core and Imperative Shell via passing values instead of relying on interfaces, developers can mitigate the risk of false positives when using mocks and encountering errors with the real implementation.
 
 ## How to Improve
 
-### [Host A Viewing Party](/practices/host-a-viewing-party.md)
+<ed> Ditto comment in other Practices about renaming this section to be a bit more specific (e.g., "How to Implement This Practice" or "Tips for Implementing This Practice"). </ed> 
 
-#### [Boundaries](https://www.destroyallsoftware.com/talks/boundaries)
+Help your team implement this practice by hosting a viewing party of specially curated talks, starting a book club to discuss impactful titles on this topic, or facilitating a roundtable discussion about relevant topics.
 
-This talk is about using simple values (as opposed to complex objects) not just for holding data, but also as the boundaries between components and subsystems. It moves through many topics: functional programming; mutability's relationship to OO; isolated unit testing with and without test doubles; and concurrency, to name some.
 
-#### [Moving IO to the edges of your app: Functional Core, Imperative Shell](https://www.youtube.com/watch?v=P1vES9AgfC4)
+### [Host a Viewing Party](/practices/host-a-viewing-party.md)
+
+#### [Boundaries by Gary Bernhardt](https://www.destroyallsoftware.com/talks/boundaries)
+
+This talk is about using simple values (as opposed to complex objects) not just for holding data but also as the boundaries between components and subsystems. It moves through many topics: functional programming, mutability's relationship to OOP, isolated unit testing with and without test doubles, concurrency, and more.
+
+#### [Moving I/O to the edges of your app: Functional Core, Imperative Shell](https://www.youtube.com/watch?v=P1vES9AgfC4)
 
 Modern architectures (such as Onion, Clean and Hexagonal) recommend that interfacing with the outside world be done at the boundaries of your app, not in the middle. Similarly, in functional programming, the core code should be deterministic, and all I/O should be at the edges. But how can you actually do this in practice? How can you separate I/O from business logic in an elegant way? In this talk, we'll look at some concrete examples of how to refactor code in this way. We'll also talk about how doing this improves code comprehension, testing, and refactoring.
```

Comment:
Looks like this was pulled directly from the youtube description. Can you paraphrase this?